.github/workflows/publish-provider-update.yml

```
@@ -35,12 +65,46 @@ jobs:
           github_token: ${{ secrets.PULUMI_BOT_TOKEN }}
           provider_short_name: ${{ env.PROVIDER_SHORT_NAME }}
           provider_version: ${{ env.PROVIDER_VERSION }}
+
+  build-remote-provider-docs:
+    if: ${{ github.event.event_type == remote-provider }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Validate Inputs
+        run: |
+          if ![ "${{ env.PROVIDER_VERSION }}     = "" ]; then echo "PROVIDER_VERSION is not respected; exit 1;    fi
+          if ![ "${{ env.PROVIDER_SCHEMA_PATH }} = "" ]; then echo "PROVIDER_VERSION is not respected; exit 1;    fi
```

```suggestion
          if ![ "${{ env.PROVIDER_SCHEMA_PATH }} = "" ]; then echo "PROVIDER_SCHEMA_PATH is not respected; exit 1;    fi
```

Also I don't speak fluent bash but that doesn't look right to me with the placement of the quotes.

I also can't tell without context what are we asserting. If I ran into this error, I wouldn't know what do do next.

---

> Also I don't speak fluent bash but that doesn't look right to me with the placement of the quotes.

There were some missing quotes. 

> I also can't tell without context what are we asserting. If I ran into this error, I wouldn't know what do do next.

These operate mostly as asserts. I've made them a bit clearer.